widgets/src/lib/InferenceWidget/widgets/AutomaticSpeechRecognitionWidget/AutomaticSpeechRecognitionWidget.svelte

```diff
@@ -141,6 +141,7 @@
 					classNames="mt-1.5"
 					{onRecordStart}
 					onRecordStop={onSelectFile}
+					bind:error
```

Comment:
Really not a fan of binding to components internal values. 

Event the Svelte doc warns against it :
```
Use component bindings sparingly. It can be difficult to track the flow of data around your application if you have too many of them, especially if there is no 'single source of truth'.
```
cf https://svelte.dev/tutorial/component-bindings

I would much rather have an explicit `onError` function prop.